### DISCUSSION TITLE
Use of `fill` when rasterizing categorical data

### DISCUSSION THREAD

**Discussion by gtmaskall** [corteva/geocube#151] [General]
The default behaviour of `make_geocube` with categorical data is to append a `nodata` string to the `<varname>_categories` variable. This makes sense from a Python indexing perspective inasmuch as the corresponding integer code for this is -1. This seems to override the `fill` argument. Again this makes sense with the use of -1 and the position of `nodata`. But if I'm correct in this, the documentation could be tightened up a little to clarify that `fill` only applies to numerical data.

**Comment by snowman2** [comment]
Thank you for catching this. A PR with a documentation update is welcome.

**Comment by gtmaskall** [comment]
First wanted to check my understanding was correct. I'll try to find time.
